user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,464 @@
+.. _admin-capabilities:
+
+Capabilities List
+=================
+
+The *"Capabilities"*  are an important part of the system's access and permission settings, as described in the 
+:ref:`Access and Permissions Basics guide <user-guide-user-management-permissions-basic>`. 
+
+Depending on the capabilities assigned to the roles, the users will have (or have no) access to different 
+functionalities. For example, you can limit the users' abiltiy to assign tags, share grids, or create new users.
+
+For each role and capability pair, there are only two options:
+
+- **None**: users with this role won’t be able to use the relevant functionality.
+- ***System***: users with this role will be able to use the relevant functionality for all the records created within 
+  the OroCRM instance they’ve logged into..
+
+  
+.. image:: ./img/roles/01.png   
+  
+This guide describes, in alphabetical order, the set of capabilities that are available and their related 
+functionalities, and provides examples of several default capability settings for some common roles.
+
+
+
+.. _admin-capabilities-acc:
+
+Abandoned Cart Campaign Manipulations
+-------------------------------------
+
+If this capability is set to "System," the user can use the *Abandoned Cart Campaign* functionality. With the functionality
+you can generate automatic e-mailings to customers in your Magento store who have not completed their purchase in order to 
+convince them to complete the checkout and place an order. You can find more details about this functionality in the
+:ref:`Magento Abandoned Cart Campaigns guide <user-guide-acc>`.
```

Comment:
Edit: If this capability is set to "System," the user can use the *Abandoned Cart Campaign* functionality. This will allow you to generate automatic e-mails to customers in your Magento store. For instance, you can send messages to customers who have not completed their purchase in order to convince them to complete the checkout and place an order. You can find more details about this functionality in the :ref:`Magento Abandoned Cart Campaigns guide <user-guide-acc>`.